src/Features/Core/Portable/FindUsages/DefinitionItem.DocumentLocationDefinitionItem.cs

```diff
@@ -37,34 +38,49 @@ internal sealed class DefaultDefinitionItem : DefinitionItem
             {
             }
 
+            private async Task<DocumentSpan?> TryGetDocumentSpanAsync(CancellationToken cancellationToken)
```

Comment:
❔ Why not just call `SourceSpans[0].RehydrateAsync`?

Reply:
so that one presumes the doc must exist.  hwoever, because we are using .CurrentSolution (and not the original solution the DefItem was created against), it could fail.  I will doc accordingly.

Reply:
💡 Consider creating a second method `TryRehydrateAsync` so the two approaches are adjacent.

Reply:
remoevd.